### PR TITLE
Fix weird results when combining -script and -e

### LINKED ISSUE #712: --execute does not work as expected
When I run 

```shell
mathics -e "Print[1+1];" -script myfile.mathics
```

where `myfile.mathics` simply contains the line `Print["Hello"];` I get

```shell
In[1]:= Print[1+1];
6
```

And `myfile.mathics` does *not* execute. This seems counter to the information printed with `mathics -h`. What *should* happen and what I would like to be able to do is that `Print[1+1];` should be executed and then `myfile.mathics` should be run as a script.

I'm happy to help fix this if I can figure out where this functionality is being implemented. Can you point me to the right part of the code?

### PR DESCRIPTION
**Pull request by TiagoCavalcanteTrindade**
Fixes #712.

### CODE REVIEW

File: test/data/script.m [hunk 1]
```diff
@@ -0,0 +1 @@
+Print["Hello"];
```
Comment (rocky) [inline test/data/script.m:1]: It would be awesome if you could get your editor to add newlines at the end of a file.

**Review by rocky — APPROVED** [review]
LGTM

File: test/data/script.m (hunk 1 above)
Comment (TiagoCavalcanteTrindade) [reply test/data/script.m:1]: I forgot to format the file.

**Comment by TiagoCavalcanteTrindade** [comment]
Looks like that `capture_output` was added in Python 3.7.

**Comment by rocky** [comment]
> 
> 
> Looks like that `capture_output` was added in Python 3.7.

I guess we can skip the test when Python < 3.7.

**Comment by TiagoCavalcanteTrindade** [comment]
Ok, how I do it?

I thought in jump the assert if the version (which I was going to get by `os` module) is smaller than 7, but I guess there's a better way.

**Comment by rocky** [comment]
> Ok, how I do it?
> 
> I thought in jump the assert if the version (which I was going to get by `os` module) is smaller than 7, but I guess there's a better way.

See 

https://stackoverflow.com/questions/38442897/how-do-i-disable-a-test-using-pytest#:~:text=Skipping%20a%20test%20based,or%20higher%22)%0Adef%20test_func_one()%3A%0A%20%20%20%20...

**Comment by TiagoCavalcanteTrindade** [comment]
@rocky maybe skep if is Windows?

File: test/test_cli.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    return osp.realpath(filename)
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason="requires Python 3.7 or higher")
+def test_cli():
+    script_file = osp.join(get_testdir(), "data", "script.m")
+
+    # asserts output contains 'Hello' and '2'
+    assert re.match(
+        r"Hello[ \n\t]*2",
```
Comment (rocky) [inline test/test_cli.py:21]: On WIndows `\r` is needed, but probably better to use `\s`

Comment (TiagoCavalcanteTrindade) [reply test/test_cli.py:21]: Yes, makes sense.

**Pull request merged**